messenger.rst

```diff
@@ -1897,6 +1897,130 @@ on a case-by-case basis via the :class:`Symfony\\Component\\Messenger\\Stamp\\Se
     provides that control. See `SymfonyCasts' message serializer tutorial`_ for
     details.
 
+Running Commands And External Processes
+---------------------------------------
+
+Trigger a Command
+~~~~~~~~~~~~~~~~~
+
+It is possible to trigger any command by dispatching a
+:class:`Symfony\\Component\\Console\\Messenger\\RunCommandMessage`. Symfony
+will take care of handling this message and execute the command passed
+to the message parameter::
+
+    use Symfony\Component\Console\Messenger\RunCommandMessage;
+    use Symfony\Component\Messenger\MessageBusInterface;
+
+    class CleanUpService
+    {
+        public function __construct(private readonly MessageBusInterface $bus)
+        {
+        }
+
+        public function cleanUp(): void
+        {
+            // Long task with some caching...
+
+            // Once finished, dispatch some clean up commands
+            $this->bus->dispatch(new RunCommandMessage('app:my-cache:clean-up'));
+            $this->bus->dispatch(new RunCommandMessage('cache:clear'));
+        }
+    }
+
+You can configure the behavior to adopt if something goes wrong during command
+execution. To do so, you can use the ``throwOnFailure`` and ``catchExceptions``
+parameters when creating your instance of
+:class:`Symfony\\Component\\Console\\Messenger\\RunCommandMessage`.
+
+Once handled, the message will contain many useful information such as the exit
```

Comment:
```suggestion
Once handled, the message result will contain many useful information such as the exit
```
?

Reply:
I rephrased to mention `RunProcessContext` and `RunCommandContext`